firedrake/mg/mesh.py

```diff
@@ -9,6 +9,11 @@
 from firedrake.cython import mgimpl as impl
 from .utils import set_level
 
+try:
+    import netgen
+except ImportError:
+    netgen = None
+    ngsPETSc = None
```

Comment:
Shouldn't there be an `import ngsPETSc` inside the `try:`?

Reply:
It should !